ArcGISRuntimeSDKQt_CppSamples/Scenes/DisplayScenesInTabletopAR/arcgisruntime.pri

```diff
@@ -0,0 +1,27 @@
+#-------------------------------------------------
+#  Copyright 2019 ESRI
+#
+#  All rights reserved under the copyright laws of the United States
+#  and applicable international laws, treaties, and conventions.
+#
+#  You may freely redistribute and use this sample code, with or
+#  without modification, provided you include the original copyright
+#  notice and use restrictions.
+#
+#  See the Sample code usage restrictions document for further information.
+#-------------------------------------------------
```

Comment:
@ldanzinger Does this need changing to the apache copyright header as well?

Reply:
yep. we will need a follow up issue to fix other files